scripts/configure_dev.sh

```diff
@@ -16,6 +16,9 @@ function install_or_upgrade {
 if [ "${OS}" = "linux" ]; then
     sudo apt-get update
     sudo apt-get -y install libboost-all-dev expect jq
+    echo "deb [trusted=yes] https://dl.bintray.com/go-swagger/goswagger-debian ubuntu main" | sudo tee -a /etc/apt/sources.list
```

Comment:
This is going to append every time someone runs configure_dev.sh.  Can we please ensure it's idempotent?

Reply:
good point

Reply:
And do this as part of the above block - update sources.list if necessary, then update, then one install set.